packages/components/psammead-play-button/README.md

```diff
@@ -0,0 +1,75 @@
+# ⛔️ This is an alpha component ⛔️
+
+This component is currently tagged as alpha and is not suitable for production use. Following the passing of an accessibility review this component will be marked as ready for production and the alpha tag removed.
+
+# psammead-play-button - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-play-button%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-play-button%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-play-button)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-play-button) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-play-button)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-play-button&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/play-button/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/play-button--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-play-button.svg)](https://www.npmjs.com/package/@bbc/psammead-play-button) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `PlayButton` component renders a button with a 'video' or 'audio' icon and optional duration timestamp. Which icon is displayed is defined by the value of the `type` prop. The component displays a 'video' icon by default. This component has options for displaying a duration timestamp, which if not provided will render gracefully without. When clicked, the function passed to `onClick` will be called.
+
+## Installation
+
+`npm install @bbc/psammead-play-button`
+
+## Props
+
+<!-- prettier-ignore -->
+| Argument       | Type      | Required | Default   | Example                  |
+| -------------- | --------- | -------- | --------- | ------------------------ |
+| service        | string    | Yes      | N/A       | `'news'`                 |
+| title          | string    | Yes      | N/A       | `'Foo.'`                 |
+| onClick        | function  | Yes      | N/A       | `() => {}`               |
```

Comment:
Would `onClick` be required for someone using this open source? Feels like this prop is only required internally.

Reply:
I deliberated over this one too and with any other element probably not, but for a `<button>` I felt it was safer and made more sense to require a handler. For the edge case where the button would do nothing, a noop could be used.